### PR TITLE
Fix: code quality issues

### PR DESCRIPTION
**Pull request by withshubh**
Hi :wave: 
I ran the [DeepSource](https://deepsource.io) static analyzer on the forked copy of this repo and found some [interesting code quality issues](https://deepsource.io/gh/withshubh/salt/issues/). This PR fixed a few of them.

#### Summary of changes

- Removed redundant call
- Removed unnecessary import alias
- Removed unused imports
- Removed duplicate elements during set declaration
- ~Added .deepsource.toml~

### Commits signed with GPG?
Yes

### CODE REVIEW

File: .deepsource.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version = 1
```
Comment (s0undt3ch) [inline .deepsource.toml:1]: Please remove this config file and please submit a feature request for discussing it's addition.

We have static analyzers running on PRs, though, admittedly, they did not catch the issues found on this PR.

**Review by s0undt3ch — CHANGES_REQUESTED** [review]
Thank you for the code fixes.
The pre-commit issues need to be fixed though.

File: .deepsource.toml (hunk 1 above)
Comment (withshubh) [reply .deepsource.toml:1]: Thanks! @s0undt3ch :sparkling_heart: 

Created the issue for the same!

**Comment by withshubh** [comment]
Hey @s0undt3ch 

Can you please guide me so that I could fixing this?

lintsalt

**Comment by withshubh** [comment]
@s0undt3ch Please have a look! :eyes:

File: salt/modules/vmctl.py [hunk 2]
```diff
@@ -11,11 +11,14 @@
...
 """
 
+
 import logging
 import re
 
+# Imoprt salt libs:
 import salt.utils.path
 from salt.exceptions import CommandExecutionError, SaltInvocationError
+from salt.ext.six.moves import zip
```
Comment (s0undt3ch) [inline salt/modules/vmctl.py:22]: This port should go, since you're editing the file.

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: salt/modules/vmctl.py (hunk 2 above)
Comment (withshubh) [reply salt/modules/vmctl.py:21]: removed! @s0undt3ch :sparkles:

File: salt/modules/vmctl.py [hunk 3]
```diff
@@ -11,9 +11,11 @@
     target machine.
 """
 
+
 import logging
 import re
 
+# Imoprt salt libs:
```
Comment (s0undt3ch) [inline salt/modules/vmctl.py:18]: Sorry, this header should also be removed

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: salt/modules/vmctl.py (hunk 3 above)
Comment (withshubh) [reply salt/modules/vmctl.py]: removed!!

**Review by s0undt3ch — APPROVED** [review]

**Comment by s0undt3ch** [comment]
Thank You!

**Comment by withshubh** [comment]
@s0undt3ch pytest check is failing. How can I fix it?

**Comment by s0undt3ch** [comment]
That's a known flaky test, but I restarted the build.

**Review by Ch3LL — APPROVED** [review]

**Review by krionbsd — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Pull request merged**